config.go

```diff
@@ -42,6 +44,8 @@ var defaultConfig = Config{
 	TunaMeasureBandwidth:   false,
 	TunaMeasureStoragePath: "",
 	SessionConfig:          nil,
+	ReconnectRetries:       5,
```

Comment:
I think it's better to make default retries unlimited (0)

Reply:
Yes, modified